WordPress/Classes/ViewRelated/MeViewController.swift

```diff
@@ -300,6 +303,9 @@ class MeViewController: UITableViewController, UIViewControllerRestoration {
     // MARK: - Notification observers
 
     func refreshModelWithNotification(notification: NSNotification) {
+        // Regenerate the header view if the account has changed
+        cachedHeaderView = MeHeaderView()
```

Comment:
I'm not sure why is this needed. `reloadViewModel` should already update the existing header view